charts/datahub/subcharts/datahub-gms/templates/deployment.yaml

```diff
@@ -137,9 +148,12 @@ spec:
               value: "{{ .Values.global.sql.datasource.driver }}"
             - name: KAFKA_BOOTSTRAP_SERVER
               value: "{{ .Values.global.kafka.bootstrap.server }}"
-            {{- with .Values.global.kafka.schemaregistry.url }}
+            {{- if eq .Values.global.kafka.schemaregistry.type "INTERNAL" }}
             - name: KAFKA_SCHEMAREGISTRY_URL
-              value: "{{ . }}"
+              value: {{ printf "http://localhost:%s/schema-registry/api/" .Values.global.datahub.gms.port }}
+            {{- else }}
```

Comment:
To be consistent with the actions POD shouldn't this env var only be applied if schema registry type is `KAFKA`?One of these templates seems wrong